eng/AcquireWasiSdk.targets

```diff
@@ -0,0 +1,62 @@
+<!--
+  Import these targets to acquire the WASI SDK.
+  - Use DependsOnTargets="AcquireWasiSdk" on the target that depends on WASI SDK.
+    - By default, this target is conditioned on $(TargetsWasi).
+    - You can depend on "AcquireWasiSdkUnconditional" instead if this doesn't work for your project.
+  - Use $(RuntimeBuildWasiSdkPath) (set by "AcquireWasiSdk") to refer to the SDK root in your target.
+-->
+<Project>
+  <PropertyGroup>
+    <_WasiSdkVersion>25.0</_WasiSdkVersion>
+    <_RuntimeLocalWasiSdkPath>$([MSBuild]::NormalizeDirectory('$(ArtifactsDir)', 'wasi-sdk'))</_RuntimeLocalWasiSdkPath>    
```

Comment:
I did not think too much about this path. I can be easily convinced it should be under e. g. `src/native/` so that nuking `artifacts` doesn't need a redownload.

Reply:
I guess we could move it later, depending on experience with it